app/src/main/res/layout/fragment_setting.xml

```diff
@@ -39,12 +39,12 @@
 
     <TextView
         android:id="@+id/tv_setting_length4"
-        android:layout_width="71dp"
-        android:layout_height="31dp"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
         android:layout_margin="12dp"
         android:layout_marginLeft="16dp"
-        android:layout_marginStart="12dp"
-        android:layout_marginTop="36dp"
+        android:layout_marginStart="16dp"
```

Comment:
Ignore this please, as we aren't done adjusting UI elements. When the UI is done, we should extract hardcoded dimens into the dimens.xml file. It's good practice to extract all hardcoded Strings and dimens into strings/dimens.xml. To do that, click on "16dp", press ALT + ENTER and select "Extract dimension resource".